### DISCUSSION TITLE
📑 Project documentation / Documentación del proyecto 📑

### DISCUSSION THREAD

**Discussion by Omitg24** [Arquisoft/lomap_es3a#37] [Ideas]
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218811134-2eec75dc-a6d5-42f7-b1d6-fae6bd5ac158.png" width="640" height="320">
</p>

# EN: Discussion objective 
<p align="justify">
Here we will talk about the documentation of the project, offering help with specific parts where we are blocked or don't know how to proceed over a certain part.
</p>

# ES: Objetivo de la discusión
<p align="justify">
Aquí hablaremos sobre la documentación del proyecto, ofreciendo ayuda con partes específicas donde estamos bloqueados o no sabemos cómo proceder sobre una parte determinada.
</p>

**Comment by Omitg24** [comment]
# Duda sobre generación de modelo de Dominio.
<p align="justify">
Hola equipo, como he comentado en la issue destinada, concretamente la issue #16, necesitaba ayuda con la generación del modelo de dominio, al menos para comprobar que está bien realizado y después evitar errores.
</p>

Muchas gracias!

**Reply by uo284373** [reply]
Hola Omar, lo que yo haría para desarrollar esta parte de la documentación sería identificar los distintos objetos que participan en el dominio de la aplicación (Marker, User, Map, Review...), incluyendo los principales atributos de cada uno.

De nada!

**Comment by Omitg24** [comment]
# Actualizados diagramas
Hola equipo! El otro día actualicé todos los diagramas del proyecto, os los dejo por aquí:

## Business Context
<p align="center">
    <img src="https://user-images.githubusercontent.com/91057639/230718285-b86e2d04-cd4f-49c6-9d4c-33af1e93052b.png">
</p>

## Technical Context
<p align="center">
    <img src="https://user-images.githubusercontent.com/91057639/230718290-9195adda-898f-4a9e-85a0-9c73cfa2bcbe.png">
</p>
 
## Infrastructura Nivel 1
<p align="center">
     <img src="https://user-images.githubusercontent.com/91057639/230718451-7d7a4f6e-456c-481e-87de-99128e493be1.png">
</p>


## Nivel 3
<p align="center">
     <img src="https://user-images.githubusercontent.com/91057639/230718298-43bed623-1148-4022-918d-2f7ec9c4630f.png">
</p>
